experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -5,42 +5,364 @@
+    * a new instance, then proceed to create a container.
+    */
+    it("can initialize an already initialized TinyliciousClient", async () => {
+        ResettableTinyliciousClient.init(clientConfig);
+        const containerConfig: TinyliciousContainerConfig = { id: documentId };
+        const schema: ContainerSchema = {
+            name: documentId,
+            initialObjects: {
+                map1: SharedMap,
+            },
+        };
+        const [container] = await ResettableTinyliciousClient.createContainer(containerConfig, schema);
+        await new Promise<void>((resolve, reject) => {
+            container.on("connected", () => {
+                resolve();
+            });
+        });
+        assert.notStrictEqual(container, undefined, "container is connected");
+    });
+
+    /**
+    * Scenario: test if the container can be created with an empty id in ContainerConfig
+    * and an empty name in ContainerSchema.
+    *
+    * Expected behavior: TinyliciousClient should throw an error
+    */
+    it("cannot create container with empty id in containerConfig", async () => {
+        const containerConfig: TinyliciousContainerConfig = { id: "" };
+        const schema: ContainerSchema = {
+            name: "",
+            initialObjects: {
+                map1: SharedMap,
+            },
+        };
+        const containerAndServices = ResettableTinyliciousClient.createContainer(containerConfig, schema);
+        const errorFn = (error) => {
+            assert.notStrictEqual(error.message, undefined, "TinyliciousClient error is undefined");
+            return true;
+        };
+
+        await assert.rejects(
+            containerAndServices,
+            errorFn,
+            "TinyliciousClient can create container with empty ID",
+        );
+    });
+
+   /**
+    * Scenario: test if the initialObjects that are passed in as parameters can be
+    * returned by the container.
+    *
+    * Expected behavior: initialObjects can be retrieved from the container.
+    */
+    it("can use container to retrieve initial objects", async () => {
+        const containerConfig: TinyliciousContainerConfig = { id: documentId };
+        const schema: ContainerSchema = {
+            name: documentId,
+            initialObjects: {
+                map1: SharedMap,
+            },
+        };
+        const [containerCreate] = await ResettableTinyliciousClient.createContainer(containerConfig, schema);
+        await new Promise<void>((resolve, reject) => {
+            containerCreate.on("connected", () => {
+                resolve();
+            });
+        });
+        const [containerGet] = await ResettableTinyliciousClient.getContainer(containerConfig, schema);
+        const initialObjectsCreate = containerCreate.initialObjects;
+        const initialObjectsGet = containerGet.initialObjects;
+        assert.notStrictEqual(initialObjectsCreate, initialObjectsGet, "container can't connect with initial objects");
```

Comment:
this check isn't useful because they will never be the same (they will not be the same reference).  it's also not actually verifying the initialObjects has the expected contents

Reply:
it's probably fine to remove this test since the one below covers what this one is trying to do anyway